src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -233,6 +265,14 @@ ReplicateActionResponse compute(String chainTaskId) {
                     context, chainTaskId);
         }
 
+        if (taskDescription.isTeeTask()) {
+            TeeService teeService = teeServicesManager.getTeeService(taskDescription.getTeeEnclaveProvider());
+            if (!teeService.prepareTeeForTask(chainTaskId)) {
+                return getFailureResponseAndPrintError(LAS_START_FAILED,
```

Comment:
```suggestion
                return getFailureResponseAndPrintError(TEE_TASK_PREPARATION_FAILED,
```
? (or better if you have)

Reply:
I think I'd prefer `TEE_PREPARATION_FAILED`. We're not preparing the task here, only the external TEE service(s).

Reply:
Better yes :)